### ISSUE TITLE
error in code

### ISSUE THREAD

**Issue by virasith05** [VSrihariMoorthy/Electronic-Voting-Machine#1]
i am unable to run the proper simualtions

**Comment by VSrihariMoorthy** [comment]
What issue are you facing? @virasith05

**Comment by virasith05** [comment]
yeah like when i am running the simulations in proteus, without pressing the push button its automatically taking its own input values
like when 1 press push button for 1st candidate its not counting and majority times it was taking as 2 as its output
even though i am not pressing for candidate 2
